share/spice/people_in_space/people_in_space.js

```diff
@@ -17,6 +17,29 @@
             };
             obj.templates.item = 'base_item';
             obj.meta.itemType = number === 1 ? "Person in Space" : "People in Space";
+            
+            obj.onItemShown = function(item) {
+                // Get filename from biophoto image url. Replace file extention jpg to png
+                // Use DDG.get_asset_path to get full path to images under spice directory
+                var imageFilename = item.biophoto.replace(/^.*[\\\/]/, ''),
+                    localImage = DDG.get_asset_path('people_in_space', "images/" + imageFilename)
+                
+                // Perform a HTTP GET request to retrieve local images using filename from API 
+                // This is used to determine if a local image exists 
+                $.ajax({
```

Comment:
Using [`$.get`](http://api.jquery.com/jQuery.get/) is simply a shorthand method for the [`$.ajax`](http://api.jquery.com/jQuery.ajax/) method.

The reason I don't fire on the `error` callback is because DuckDuckGo production will not return a non 200 HTTP status code, essentially meaning it's a success. Example https://duckduckgo.com/share/spice/people_in_space/1177/image.png - [see status code returned](https://gist.githubusercontent.com/MrChrisW/4d941d83bd9fee542990/raw/5487baa7d70c47c0cf1bf7f69d10c66336a1817e/example.sh). 

Also... I've tested using the dataType parameter at the end of the $.get function like in your example above and am unable to get it to function correctly.